scene/main/viewport.cpp

```diff
@@ -305,7 +305,12 @@ void Viewport::_sub_window_grab_focus(Window *p_window) {
 	}
 
 	if (old_focus) {
-		_sub_window_update(old_focus);
+		Viewport *embedder = old_focus->_get_embedder();
+		if (embedder) {
+			embedder->_sub_window_update(old_focus);
```

Comment:
I hope I didn't think too much, considering that there may be multiple levels of embeddings.